node/src/Pos/Launcher/Scenario.hs

```diff
@@ -72,11 +72,9 @@ runNode' NodeResources {..} workers' plugins' = ActionSpec $ \vI sendActions ->
     nodeStartMsg
     inAssertMode $ logInfo "Assert mode on"
     pk <- getOurPublicKey
-    addr <- getOurPubKeyAddress
     let pkHash = addressHash pk
-    logInfoS $ sformat ("My public key is: "%build%
-                        ", address: "%build%
-                        ", pk hash: "%build) pk addr pkHash
+    logInfoS $ sformat ("My public key is: "%build%", pk hash: "%build)
```

Comment:
https://github.com/input-output-hk/cardano-sl/commit/0c375b03eb0058bbd4d6cfee21b6f6b25b5545c6

I also don't want to print address depending on whether we are in bootstrap era, because it requires blocking. The only thing I can do is to print two addresses.